.github/workflows/build-test-lint.yaml

```diff
@@ -16,11 +16,12 @@ on:
 
 jobs:
   build:
```

Comment:
Shana merged a PR to test/resolve pipeline issues (i should've done it separately... apologies!)